third_party/blink/renderer/platform/media/BUILD.gn

```diff
@@ -84,7 +84,7 @@ component("media") {
     "//third_party/blink/renderer/platform:allow_discouraged_type",
   ]
 
-  if (media_use_ffmpeg || !is_android) {
+  if (media_use_ffmpeg || !is_android || is_cobalt) {
```

Comment:
Why do we need these sources if we don't use ffmpeg?

Reply:
We need web_media_player_impl* specifically. The check for this is weird in that it builds for all platforms when `media_use_ffmpeg` is set but doesn't for android when it's not. We can make a more granular change and take exactly what we need, I just wanted to keep the change small.

Reply:
I see what you mean, it does seem like something that needs to be changed upstream